ParisBNF/abb/BNFabb92.xml

```diff
@@ -0,0 +1,682 @@
+                    >Ḫǝdār</foreign></title>
+                <title xml:lang="gez" xml:id="t2">መጽሐፈ፡ ኅዳር፡</title>
+                <title xml:lang="gez" corresp="#t2" type="normalized">Maṣḥafa Ḫǝdār</title>
+                <title xml:lang="fr" xml:id="t3">Melanges théologiques </title>
+                <title xml:lang="fr" xml:id="t4" resp="PRS3082Chaine">Récits
+                    hagiographiques.</title>
+                <editor key="NV"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine
+                    multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                        <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b531387571">BnF Éthiopien
+                            d'Abbadie 92</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1r" to="16rb"/>
+                            <title type="complete" xml:lang="en" ref="LIT6168ManifOfHab"
+                                >Manifestation of the prophet <persName ref="PRS4914Habakkuk"
+                                    >Habakkuk</persName> at Qarṭasā (<date ref="ethiocal:Hedar3">3rd
+                                    of Ḫǝdār</date>)</title>
+                            <incipit xml:lang="gez">
+                                <locus from="1ra"/><hi rend="rubric">አመ፡ ሠሉሰ፡ ለኅዳር፡</hi> አስተርእዮቱ፡
+                                    ለአን<hi rend="rubric">ባቆም፡ ነቢይ፡ በአ</hi>ማን፡ በሃገረ፡ እንተ፡ ስ<hi
+                                    rend="rubric">ማ፡ ቀርጠሳ፡ እንዘ፡ ሊቀ፡ ጳጳሳት፡ አባ፡ አርሚስ፡</hi>
+                            </incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus target="#16rb"/>
+                            <title type="complete" xml:lang="en" ref="LIT6170GadlaEleazar">
+                                Commemoration of <persName ref="PRS13141Eleazar">Eleazar</persName>,
```

Comment:
Is it a gadl or a commemorations?

Reply:
> Is it a gadl or a commemorations?

it is very short and Marcin called the text ´commemoration´ in English, also the word Gadl is mentioned. I think commemoration fits.